src/document.cpp

```diff
@@ -357,6 +357,11 @@ int litehtml::document::render( int max_width, render_type rt )
 	int ret = 0;
 	if(m_root)
 	{
+		litehtml::position::vector
+			redraw_box_table;
+
+		m_root->find_styles_changes( redraw_box_table, 0, 0 );

```

Comment:
shouldn't you only do this if the dirty flag is set? + It would be great if we can use the `redraw_box_table`. That might be a bit more work though